packages/webapp/src/components/Modals/WaterUsageCalculatorModal/index.jsx

```diff
@@ -38,43 +37,31 @@ const WaterUseVolumeCalculator = ({ system, setTotalWaterUsage, totalWaterUsage,
   const {
     estimated_flow_rate,
     estimated_flow_rate_unit,
-    estimated_irrigation_duration,
-    estimated_irrigation_duration_unit,
-  } = getValues();
+    estimated_duration,
+    estimated_duration_unit,
+  } = getValues().irrigation_task;
 
   useEffect(() => {
-    if (estimated_flow_rate && estimated_irrigation_duration) {
+    if (estimated_flow_rate && estimated_duration) {
       setTotalWaterUsage(() => {
-        if (
-          estimated_flow_rate_unit.label === 'l/h' &&
-          estimated_irrigation_duration_unit.label === 'h'
-        )
-          return estimated_flow_rate * estimated_irrigation_duration;
-        if (
-          estimated_flow_rate_unit.label === 'l/m' &&
-          estimated_irrigation_duration_unit.label === 'min'
-        )
-          return estimated_flow_rate * estimated_irrigation_duration;
-        if (
-          estimated_flow_rate_unit.label === 'l/h' &&
-          estimated_irrigation_duration_unit.label === 'min'
-        )
-          return (estimated_flow_rate * estimated_irrigation_duration) / 60;
-        if (
-          estimated_flow_rate_unit.label === 'l/m' &&
-          estimated_irrigation_duration_unit.label === 'h'
-        )
-          return estimated_flow_rate * estimated_irrigation_duration * 60;
+        if (estimated_flow_rate_unit.label === 'l/h' && estimated_duration_unit.label === 'h')
+          return estimated_flow_rate * estimated_duration;
+        if (estimated_flow_rate_unit.label === 'l/m' && estimated_duration_unit.label === 'min')
+          return estimated_flow_rate * estimated_duration;
+        if (estimated_flow_rate_unit.label === 'l/h' && estimated_duration_unit.label === 'min')
+          return (estimated_flow_rate * estimated_duration) / 60;
+        if (estimated_flow_rate_unit.label === 'l/m' && estimated_duration_unit.label === 'h')
+          return estimated_flow_rate * estimated_duration * 60;
```

Comment:
estimated_flow_rate and estimated_duration should give values in irrigation_task_estimated_duration.databaseUnit and irrigation_task_estimated_flow_rate.databaseUnit.
Something might have gone seriously wrong

Reply:
@JimmyRowland this has been resolved